nil/client/rpc/client.go

```diff
@@ -969,3 +971,11 @@ func simpleCallUint64[ReturnType ~uint64](
 	}
 	return ReturnType(result), err
 }
+
+func (c *Client) GetTxpoolStatus(ctx context.Context, shardId types.ShardId) (uint64, error) {
+	return simpleCallUint64[uint64](ctx, c, Txpool_getTxpoolStatus, shardId)
+}
+
+func (c *Client) GetTxpoolContent(ctx context.Context, shardId types.ShardId) ([]*types.Transaction, error) {
```

Comment:
```suggestion
func (c *Client) GetTxpoolContent(ctx context.Context, shardId types.ShardId) (jsonrpc.TxPoolContent, error) {
```

Reply:
Seems this function is untested...

Reply:
I add simple test just for call. Now I don't really understand how to stop pool to handle transaction. Maybe it's a good todo for next PR.